src/Integration/Binding/SolutionBindingOperation.cs

```diff
@@ -255,7 +263,7 @@ private void PendBindingInformation(ConnectionInformation connInfo)
                 connInfo.Organization);
             bound.Profiles = map;
 
-            // TODO: choose solution binding mode
+            // TODO: duncanp choose solution binding mode
```

Comment:
The binding mode is being supplied in the constructor, so the next line should be using it. It'll be part of the next PR.